### PR TITLE
Fix how to add Julia releases with juliaup

### PR DESCRIPTION
**Pull request by michmx**
Hi. I am following the tutorial and noticed that the instruction to add new releases with juliaup doesn't work. Looking at the [juliaup repo](https://github.com/JuliaLang/juliaup), it indicates the command to install Julia releases is 
```
juliaup add <release>
```

I am a beginner con Julia, so please check if I am missing something. :)

I am also updating the doc to the most recent Julia release and commenting how to keep juliaup updated.

### CODE REVIEW

File: julia-intro/docs/julia-intro-installation.ipynb [hunk 1]
```diff
@@ -138,15 +138,21 @@
     "\n",
     "to show all the versions of Julia available.\n",
     "\n",
-    "We will work with the newest version on all platforms, 1.9.3 so simply type\n",
+    "We will work with the newest version on all platforms, 1.10.4 so simply type\n",
     "\n",
     "```bash\n",
-    "juliaup install 1.9.3\n",
+    "juliaup add 1.10.4\n",
```
Comment (Moelf) [inline julia-intro/docs/julia-intro-installation.ipynb:144]:
```suggestion
    "juliaup add release\n",
```

File: julia-intro/docs/julia-intro-installation.ipynb [hunk 2]
```diff
@@ -138,15 +138,21 @@
     "\n",
     "to show all the versions of Julia available.\n",
     "\n",
-    "We will work with the newest version on all platforms, 1.9.3 so simply type\n",
+    "We will work with the newest version on all platforms, 1.10.4 so simply type\n",
```
Comment (Moelf) [inline julia-intro/docs/julia-intro-installation.ipynb:141]:
```suggestion
    "We will work with the latest `release` version on all platforms (1.10.4 as of writing) so simply type\n",
```

Comment (michmx) [reply julia-intro/docs/julia-intro-installation.ipynb]: Thanks @Moelf !

**Comment by Moelf** [comment]
note to whoever is gonna update for thsi year's JuliaHEP: we should point people to https://modernjuliaworkflows.github.io/writing/#installation for this part (and other parts of the basic setup and tutorial whenever applicable)

**Pull request merged**